src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/PreferIsEmptyOverCountTests.cs

```diff
@@ -435,6 +435,22 @@ public Task CSharpTestWhitespaceTrivia()
         ? 0 :
         _dictionary.Count;
 }");
+
+        [Theory]
+        [InlineData("System.ReadOnlyMemory")]
+        [InlineData("System.ReadOnlySpan")]
+        [InlineData("System.Memory")]
+        [InlineData("System.Span")]
+        public Task CSharpTest_DisallowedTypesForCA1836_NoDiagnosis(string type)
```

Comment:
[minor] Feel free to ignore but it seems that "most" tests seem to use the `_NoDiagnostic` suffix instead.

Reply:
This file in particular uses `NoDiagnosis` for all other tests so for now I will keep it as is.

Reply:
Oh sorry about that.